src/main/resources/resource/Deeplearning4j/Deeplearning4j.py

```diff
@@ -7,3 +7,12 @@
 
 # start the service
 deeplearning4j = Runtime.start('deeplearning4j','Deeplearning4j')
+
+# load the VGG16 model from the zoo
+deeplearning4j.loadVGG16()
+
+# run an image file through the model and get the classifications / confidence
+classifications = deeplearning4j.classifyImageFileVGG16("myimage.jpg")
+
+# print them out... it's a dictionary/map
+print classifications
```

Comment:
its 2.7 Jython but shouldn't we always print with parens since it supports it ?

Reply:
old habits.  i'll clean it up.

Reply:
updated.